test-projects/integration/traces.visivo.yml

```diff
@@ -99,7 +99,7 @@ traces:
       type: scatter3d
       x: query(x)
       y: query(y)
-      z: query( -2*(x%2)*x + x + y*(y-3))
+      z: query( -2*(x/16)*x + x + y*(y-3))
```

Comment:
Why did this change?

Reply:
Big query doesn't support the `%` as a modulus. The logic was arbitrary anyways so I switched it so we could easily add another integration test for bigquery